console/web3ext/web3ext.go

```diff
@@ -899,12 +907,24 @@ web3._extend({
 			params: 1,
 			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
 		}),
+		new web3._extend.Method({
+			name: 'getParams',
+			call: 'klay_getParams',
+			params: 1,
+			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
+		}),
 		new web3._extend.Method({
 			name: 'chainConfigAt',
 			call: 'klay_chainConfigAt',
```

Comment:
It's minor comment.
I think It's better to remove `itemsAt` and `chainConfigAt` function.
And then we can add alias to point itemsAt --> getParams, chainConfigAt --> getChainConfig until mantle.

You can ignore this comment if you think it's not important.

Reply:
Do you mean removing `func (api *PublicGovernanceAPI) ItemsAt` and fixing itemsAt like below?
```
		new web3._extend.Method({
			name: 'itemsAt',
			call: 'governance_getParams',
			params: 1,
			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
		}),
```

This will remove `governance_itemsAt` API.
However, as stated in PR description itemsAt and chainConfigAt are to be removed in v1.11.

Reply:
I understood but this suggestion is more clear when you work in mantle in the future I think. It's minor. Thanks for explaining about it.